etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
I still don't understand how other people are going to use this for real. It is confusing.
Papertrail is (normally) a command to work with the papertrail logging-as-a-service thingy. Only production heroku logs will be there. 

This would never work in production because webservers are not going to have papertrail tokens. 

This log endpoint is just for debugging, which is fine. But having the debug script here "look" like the production log tailer is confusing to me.